### ISSUE TITLE
The readme file does not describe how to build the site the first time.

### ISSUE THREAD

**Issue by ecnalyr** [ghoppe/refinerycms-bootstrap#1]
Maybe I'm missing something, but I cannot (in 10 mins time) get the rails server command to work on my local machine.

**Comment by ghoppe** [comment]
This gem is a small engine for [RefineryCMS](https://github.com/refinery/refinerycms) to add Sass flavoured Bootstrap to the views. It's meant to be added to an existing RefineryCMS installation. Apologies for the lack of documentation, it's just a simple thing I'm using for my own projects.

If you need help getting RefineryCMS up and running, see the [Refinery Guide](http://refinerycms.com/guides) pages at refinerycms.com. If you're still having trouble, there's often helpful people in the #refinerycms IRC channel on freenode.

**Comment by ecnalyr** [comment]
My apologies.  I thought this was intended to be a stand-alone app.  I didn't identify it as a gem as I am still new to the rails world.

Thank you for the clarification.

**Comment by ecnalyr** [comment]
I still feel I personally could benefit from a little more instruction on how to get started using your gem.

I have included your gem using: gem 'refinerycms-bootstrap', :git => 'git://github.com/ghoppe/refinerycms-bootstrap.git'
 within my gem file.

I run a bundle then rakedb:migrate on my previously fully functional 'starter RefineryCms app' and can launch the app but do not know where to begin to get to where I can view what your gem actually does.

Could you point me in the direction I need to go that allows me to quickly see what your gem can do?

I thank you for your time.

**Comment by ghoppe** [comment]
Certainly, this basic information should really be in the docs sometime, as I said I made this for myself, and someone not familiar with Rails wouldn't immediately know the magic that goes on behind the scenes.

The only thing I say in the docs is that this gem uses Thomas McDonald's [bootstrap-sass](https://github.com/thomas-mcdonald/bootstrap-sass) to give RefineryCMS basic Bootstrap functionality. You should follow the docs at  [bootstrap-sass](https://github.com/thomas-mcdonald/bootstrap-sass) to setup your CSS/javascript for the project. Briefly, this is as simple as adding the following to your `/app/assets/application.js`:

```
//= require bootstrap
```

Then create a new css or scss file in `/app/assets/stylesheets` (I call it `myappname.scss` or whatever) and include the following:

```
@import "bootstrap"
```

This file should automatically get loaded in a basic rails/refinerycms install. This will import all the CSS styles and Javascript for Bootstrap. Then you can use all the [Bootstrap CSS/Javascript goodies](http://twitter.github.com/bootstrap/) in the web views for your RefineryCMS/rails application.

The difference between using this gem and bootstrap-sass is that my gem includes a few replacement views for the RefineryCMS administrative backend to use Bootstrap button styles etc. — that part is still pretty rough and needs some more tweaking for consistency. I also include a few front-end tweaks for a quick Bootstrap menu replacement, but that's all it does! 

In short, this gem just makes it easy to plug [bootstrap-sass](https://github.com/thomas-mcdonald/bootstrap-sass) into a [RefineryCMS](https://github.com/refinery/refinerycms) application for interface consistency and customization.

**Comment by ecnalyr** [comment]
Works wonderfully.  Thank you for the gem!

**Comment by vartanian** [comment]
Would you say it is better to install refinery before or after the bootstrap gem?
